context/app/static/js/components/detailPage/files/DataProducts/hooks.ts

```diff
@@ -16,20 +16,42 @@ const defaultPipeline: PipelineInfo = {
   origin: 'https://github.com/hubmapconsortium/ingest-pipeline.git',
 };
 
+// Values that should not be displayed as pipeline info
+const originBlacklist = ['https://github.com/hubmapconsortium/portal-containers'];
+
+// Undescriptive pipeline names which should be replaced with GitHub repo names
+const nameBlacklist = ['pipeline.cwl'];
+
+function getGithubRepoName(origin: string) {
```

Comment:
Perhaps add a unit test?

Reply:
I added test cases for this function and pulled out the `.reduce` function as well to make it easier to test in isolation 👍🏻